x-pack/plugins/cases/public/components/case_action_bar/index.test.tsx

```diff
@@ -70,21 +68,6 @@ describe('CaseActionBar', () => {
     expect(wrapper.find(`[data-test-subj="sync-alerts-switch"]`).exists()).toBeTruthy();
     expect(wrapper.find(`[data-test-subj="case-refresh"]`).exists()).toBeTruthy();
     expect(wrapper.find(`[data-test-subj="case-view-actions"]`).exists()).toBeTruthy();
-    // no loading bar
-    expect(wrapper.find(`[data-test-subj="case-view-action-bar-spinner"]`).exists()).toBeFalsy();
```

Comment:
User actions are not used anymore in this component so I removed the loading spinner.